samples/sp-webpart/src/webparts/mgtDemo/MgtDemoWebPart.ts

```diff
@@ -25,15 +26,7 @@ export default class MgtDemoWebPart extends BaseClientSideWebPart<IMgtDemoWebPar
   }
 
   public render(): void {
-    const element = React.createElement(
-      React.Suspense,
-      {
-        fallback: React.createElement('div', null, 'Loading')
-      },
-      React.createElement(MgtDemo, {
-        description: this.properties.description
-      })
-    );
+    const element = lazyLoadComponent(MgtDemo, { description: this.properties.description });
```

Comment:
Where is the MgtDemo component?

Reply:
Imported on line 15 with a lazy:
```
const MgtDemo = React.lazy(() => import('./components/MgtDemo'));
```

Reply:
Oh. I don't see the actual files. Now I understand. I hate that I can't see all files!!!!!

Reply:
It's a side effect of me making this a PR into the disambiguation branch rather than just a commit there.